operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -282,8 +282,38 @@ This procedure will add a liquid-cooled blades from a HPE Cray EX system.
 
 14. Use boot orchestration to power on and boot the nodes. Specify the appropriate BOS template for the node type.
 
+    1. Determine how the BOS Session template references compute hosts. Typically, they are referenced by their "Compute" role. However, if they are referenced by xname, then these new nodes should added to the BOS Session template.
     ```bash
```

Comment:
The code blocks and text under each sub-step in this PR need more indentation. When this is rendered, the numbering and alignment of the code blocks is inaccurate.

Reply:
@nrockershousen , I would like to render the content ahead of time, so I can view it and see if it works. How do I do that?

Reply:
I'm not sure what tool you are using to edit markdown files, so it might vary. I use Visual Studio Code, which allows me to view a rendered preview when I open up the file and hit **ctrl+shift+v**.

Reply:
I was using emacs, which has a markdown mode, but it takes some doing to get it to work.